src/views/Home/Home.js

```diff
@@ -7,12 +8,28 @@ import './Home.css'
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
+  const [title, setTitle] = useState('')
+  const [subtitle, setSubtitle] = useState('')
+  const [font, setFont] = useState('')
```

Comment:
Great job on this one! Remember that you can use default values for your setState() calls. This comes in handy for properties like font and align.